packages/integrations/vercel/test/fixtures/serverless-prerender/astro.config.mjs

```diff
@@ -4,7 +4,8 @@ import vercel from '@astrojs/vercel/serverless';
 export default defineConfig({
 	adapter: vercel({
 		// Pass some value to make sure it doesn't error out
-		includeFiles: ['included.js']
+		includeFiles: ['included.js'],
+		functionPerRoute: false,
```

Comment:
@ematipico I had to turn this off for this test. I'm not sure why, it could be a bug with functionPerRoute with prerendering.

Reply:
This should be fixed now